docs/topics/jvm/java-interop.md

```diff
@@ -137,20 +144,76 @@ Set<@NotNull String> toSet(@NotNull Collection<@NotNull String> elements) { ...
 It leads to the following signature seen in Kotlin:
 
 ```kotlin
-fun toSet(elements: (Mutable)Collection<String>) : (Mutable)Set<String> { ... }
+fun toSet(elements: (Mutable)Collection<String>) : (Mutable)Set<String>
 ```
 
 Note the `@NotNull` annotations on `String` type arguments. Without them, you get platform types in the type arguments:
 
 ```kotlin
-fun toSet(elements: (Mutable)Collection<String!>) : (Mutable)Set<String!> { ... }
+fun toSet(elements: (Mutable)Collection<String!>) : (Mutable)Set<String!>
+```
+
+Kotlin also takes into account nullability annotations on type arguments of base classes and interfaces. For example,
+there are 2 Java classes with the signatures provided below.
+
+```java
+public class Base<T>
```

Comment:
Minor: I don't know if there is a requirement for inserting Java code pieces to be valid, but if so, then here and below it is missing `{}`.

Reply:
@p7nov WDYT? Do we have such a requirement? Can I add a snippet with the signatures only?

Reply:
It's always better to have valid copy-pastable snippets. Exception: if this requires a lot of irrelevant code :)
In this case it's not a problem at all, let's add the braces.